www/tilesource-iiif.html

```diff
@@ -3,14 +3,14 @@
 </h2>
 <p>
     The latest version (1.1) of the International Image Interoperability Framework: Image API is formally described
```

Comment:
+1. I made the changes. @iangilman let me know if you would like the 
commits squashed.